### DISCUSSION TITLE
Cron Jobs stopped working

### DISCUSSION THREAD

**Discussion by devashish-sood** [vercel/community#6505] [Account Issue]
### Question

Despite no changes to the source code, my cron jobs stopped working today. Here are the contents of my `vercel.json` file: 

```
{
  "crons": [
    {
      "path": "/api/choresEvening",
      "schedule": "0 1 * * *"
    }, 
    {
      "path": "/api/choresMorning", 
      "schedule":"0 14 * * *"
    }
  ]
}
```

I've set them up to execute serverless functions once a day, and I've confirmed that the functions are not erroring out by manually hitting them on the prod url.

**Comment by aal-khadzh** [comment]
I'm facing the same issue. Cron expression has been validated and job can be run manually by clicking "Run" button within Cron Jobs section of the project settings. When run manually, there is a log input appearing. However, logs doesn't show any API call around the time cron job is scheduled.

**Reply by justingebert** [reply]
Same here :/

**Reply by justingebert** [reply]
> I'm facing the same issue. Cron expression has been validated and job can be run manually by clicking "Run" button within Cron Jobs section of the project settings. When run manually, there is a log input appearing. However, logs doesn't show any API call around the time cron job is scheduled.
@aal-khadzh 
did you solve it? I am still having the issue. Could it be something with the execution limit?

**Reply by justingebert** [reply]
turns out it was a race condition in my case

**Comment by chreds** [comment]
I have the same issue but I'm not getting logs when the job is run manually from the cron jobs settings page although I can still run the job using postman and triggering it that way.

**Comment by chreds** [comment]
I found out what happened with mine. My nextjs file had a trailing slashes setting turned on so anything without a trailing slash was being forwarded. It looks like the cron setup in vercel will not follow redirects but doesn't log an error about it. So they were silently failing.

**Reply by teddis** [reply]
Thanks for the heads up. Happening to me and my trailingSlash was set to `true`. I am trying `false`. Can I assume this solved your problem? I'm also concerned with the lack of support monitoring and feedback here.

**Reply by chreds** [reply]
Yes, setting to false fixed it. Also you can try just adding a slash to the end of your cron job function path.

To be fair I emailed support and they were able to track this down for me.

**Reply by teddis** [reply]
Beautiful. Working now for me. I submitted ticket, and replied I got this working with your solution. I hope they update their documentation. Cheers.

**Reply by jasondainter** [reply]
Hero! Thanks this was driving me crazy, adding one trailing slash to my vercel.json fixed it! 

Eg

"path": "/api/yourfunction**/**",

If you have   

trailingSlash: true, 

in your next.config.js

**Comment by jesper-bylund** [comment]
I'm also having the same issue. But it's clearly something that's happening, not a setting:
When I deploy a new version it works fine, then after a few days it stops triggering. No errors or reasons that I can find in the logs. 🤔

**Comment by muhammadkasimm** [comment]
In my case, I can't view my cron job under project Settings > Cron Jobs. My last deployment was about 300 days ago and I believe it was working fine until now, but now I can't even see it listed under Cron Jobs. Did they change something?

[a comment by an automated account is not shown]
